### PR TITLE
Turn on onnx fallthrough in convert_model()

### PR DESCRIPTION
**Pull request by popovaan**
Description: 
ONNX_FALLTHROUGH allows to export PyTorch models to ONNX with unsupported operations as is. Without this option torch.onnx.export() fails to export such models.
Use case of turning on the option - converting PyTorch models with custom user extensions.
Ticket: 98995

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR - N/A
* [x]  Transformation preserves original framework node names - N/A
* [x]  Transformation preserves tensor names - N/A

Validation:
* [x]  Unit tests - N/A
* [x]  Framework operation tests - N/A
* [x]  Transformation tests - N/A
* [x]  Model Optimizer IR Reader check - N/A
* [x]  Checked conversion on TorchVision and timm models, no regression occurred.
* [x]  Checked conversion on examples from https://github.com/openvinotoolkit/openvino_contrib/pull/481 This PR allows to convert all examples using following line:
`convert_model(model, example_input=inp, extensions="libuser_ov_extensions.so", use_legacy_frontend=True)`

Documentation:
* [x]  Supported frameworks operations list - N/A
* [x]  Guide on how to convert the **public** model - N/A
* [x]  User guide update - N/A

### CODE REVIEW

File: tools/mo/openvino/tools/mo/convert_impl.py [hunk 1]
```diff
@@ -723,6 +723,7 @@ def convert_pytorch_to_onnx(model, input_shape, opset_version, example_inputs, o
     torch.onnx.export(model,
                       inputs,
                       model_onnx,
+                      operator_export_type=torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH,
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/convert_impl.py:726]: @popovaan, @eaidova, is it possible to add one small PyTorch model with unsupported operation please?

Comment (popovaan) [reply tools/mo/openvino/tools/mo/convert_impl.py]: Added test with small custom op, which checks that conversion to ONNX step passes.

File: tools/mo/openvino/tools/mo/moc_frontend/pytorch_frontend_utils.py [hunk 2]
```diff
@@ -131,6 +131,7 @@ def convert_pytorch_to_onnx(model, input_shape, opset_version, example_inputs, o
     torch.onnx.export(model,
                       inputs,
                       model_onnx,
+                      operator_export_type=torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH,
                       **additional_params)
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/moc_frontend/pytorch_frontend_utils.py:135]: not a blocker for this PR but we need to consider `try-except` to properly handle possible exceptions.
@popovaan, can you create separate PR please for this?

**Review by rkazants — APPROVED** [review]

**Comment by rkazants** [comment]
@eaidova, can you please also take a look?

**Review by eaidova — APPROVED** [review]

File: tools/mo/openvino/tools/mo/moc_frontend/pytorch_frontend_utils.py (hunk 2 above)
Comment (popovaan) [reply tools/mo/openvino/tools/mo/moc_frontend/pytorch_frontend_utils.py:135]: Yes, I can add `try-except` is separate PR

**Pull request merged**